source/index.html.md

```diff
@@ -5417,7 +5419,8 @@ func main() {
     {
       "ref": "PR.4",
       "initiator_id": "ca7bc5b3-e47f-4153-96fb-bbe326b42772",
-      "authoriser_id": "fb6a9252-3818-44dc-b5aa-2195391a746f",
+      "authoriser_id": "de86472c-c027-4735-a6a7-234366a27fc7",
+      "authoriser_contact_id": "ca7bc5b3-e47f-4153-96fb-bbe326b42772",
```

Comment:
is it normal that initiator_id and authoriser_contact_id are identical?

Reply:
Good 👁